package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "banka",
+  "version": "1.0.0",
+  "description": "A banking application that helps users perform banking transactions.",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel server/src -d server/dist",
+    "dev": "nodemon ./server/src/app.js --exec babel-node",
+    "start": "nodemon --exec babel-node src/app.js",
```

Comment:
Why are you still starting the app with `nodemon` and `babel-node`? You should be starting the built app instead by using just `node`. Kindly update this

Reply:
noted

Reply:
do you mean this  "start": "node ./server/src/app.js"

Reply:
Can you start directly from the source folder using node?